modules/ipi-install-configuring-raid-for-worker-node.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-raid-for-worker-node_{context}"]
+= Configuring RAID for worker node
+
+The following procedure configures RAID for the worker node during the installation process.
+
+.Procedure
+. Create manifests.
+. Modify the BMH file corresponding to the worker:
++
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+----
++
+[NOTE]
+====
+Since servers with bmc type `irmc` does not support software RAID, the following RAID configuration uses hardware RAID as an example.
+====
++
+.. If a specific RAID configuration is added to the spec, this will cause the worker node to delete the original RAID configuration
+in the `preparing` phase and perform a specified configuration on the RAID. For example:
++
+----
+spec:
+  raid:
+    hardwareRAIDVolumes:
+	- level: "0"
+	  name: "sda"
+	  numberOfPhysicalDisks: 1
+	  rotational: true
+	  sizeGibibytes: 0
+----
+<1> `level` is a required field, and the others are optional fields. See the
+link:https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#raid[BMH documentation]
+for details.
++
+.. If an empty RAID configuration is added to the spec, this will cause the worker node to delete the original RAID configuration
```

Comment:
s/If an empty RAID configuration is added to the spec/if you added an empty RAID configuration to the spec

s/this will cause/this empty configuration causes

Should this line through the note below be a second-level bulleted list?

Also, remove the hard return on this line.

Reply:
>Should this line through the note below be a second-level bulleted list?

I think this is an illustration of the second case and should put here.